webpack.common.js

```diff
@@ -42,7 +42,7 @@ module.exports = {
 			{
 				test: /\.js$/,
 				loader: 'babel-loader',
-				exclude: /node_modules/,
+				exclude: /node_modules(?!(\/|\\)(@juliushaertl\/vue-richtext|fast-xml-parser|hot-patcher|nextcloud-vue-collections|webdav)(\/|\\))/,
```

Comment:
what, you use webdav & hotpatcher here?

Reply:
webdav for the file upload
hotpatcher I dont know, it was complained by IE and when I added it here it worked.

Reply:
Ah! I was not aware we had file upload yet ^^

Reply:
Sorry I was afraid you used the viewer config file ;)

Reply:
```
$ npm ls hot-patcher
talk@9.0.0 /home/nickv/Nextcloud/19/server/appsbabies/spreed
└─┬ webdav@3.2.0
  └── hot-patcher@0.5.0 
```

would it be enough to add `webdav` ?

Reply:
No, there is no nested dependencies :)